docs/source/developer/indy-ansible.md

```diff
@@ -0,0 +1,303 @@
+# Indy Configurations
+
+In the Blockchain Automation Framework project, ansible is used to automate the certificate generation, putting them in vault and generate value files, which are then pushed to the repository for deployment, using GitOps. This is achieved using Ansible playbooks. 
+Ansible playbooks contains a series of roles and tasks which run in sequential order to achieve the automation.
+
+```
+/hyperledger-indy
+|-- charts
+|   |-- indy-auth-job
+|   |-- indy-cli
+|   |-- indy-domain-genesis
+|   |-- indy-domain-genesis
+|   |-- indy-key-mgmt
+|   |-- indy-ledger-txn
+|   |-- indy-node
+|   |-- indy-pool-genesis
+|-- images
+|-- configuration
+|   |-- roles/
+|   |-- samples/
+|   |-- playbook(s)
+|   |-- cleanup.yaml
+|-- releases
+|   |-- dev/
+|-- scripts
+|   |-- indy_nym_txn
+|   |-- setup indy cluster
+```
+
+For Hyperledger-Indy, the ansible roles and playbooks are located at `/platforms/hyperledger-indy/configuration/`
+Some of the common roles and playbooks between Hyperledger-Indy, Hyperledger-Fabric and R3-Corda are located at
+`/platforms/shared/configuration/`
+
+--------
+
+## Roles for setting up Indy Network
+
+Roles in ansible are a combination of logically inter-related tasks.
+
+Below is the single playbook that you need to execute to setup complete corda network.
```

Comment:
This statement is wrong. If you want to add this, insert the deploy-network playbook description/name